cuecfg/cuecfg_test.go

```diff
@@ -0,0 +1,49 @@
+package cuecfg
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type Metadata struct {
+	Tags []string
+}
+
+type MyConfig struct {
+	Version int
+	Name    string
+	Meta    *Metadata
+}
+
+var testTomlCfg = &MyConfig{
+	Version: 2,
+	Name:    "go-toml",
+	Meta: &Metadata{
+		Tags: []string{"go", "toml"},
+	},
+}
+
+var testTomlStr = `Version = 2
```

Comment:
I find `heredoc.Doc` handy for these cases

Reply:
ah, I like that

sadly, because of gofmt it ends up like this:
```
var testTomlStr = heredoc.Doc(
        `Version = 2
        Name = 'go-toml'

        [Meta]
        Tags = ['go', 'toml']
`,
)
```

Reply:
which isn't much better IMO 🤷🏾  😭

Reply:
Are you sure?

Have you tried:
```
var testTomlStr = heredoc.Doc(`
        Version = 2
        Name = 'go-toml'

        [Meta]
        Tags = ['go', 'toml']
`)
```

(notice placement of the backticks)